### DISCUSSION TITLE
Alternative generation method using routes instead of a crawler

### DISCUSSION THREAD

**Discussion by Tofandel** [spatie/laravel-sitemap#487] [Ideas]
Why use a crawler and not a list of routes/controllers?

It would be much more performant if we could specify routes, get the controller method and get the possible models available from a reflection class and a query, or provide a manual list of options or fallback to a crawler for routes that can't be detected

**Comment by freekmurze** [comment]
That could be a valid approach too. 

You create that package 👍
